lib/form/html.js

```diff
@@ -113,10 +113,14 @@ const addRequired = (html, input) => {
 const renderer = (type, errors) => {
   return new Promise((res) => {
     let rendered = type.attributes.map(input => {
-      const inputs = Object.keys(input.inputs).length;
+      let inputs = Object.keys(input.inputs).length;
+      if (Array.isArray(input.inputs) && input.inputs.length > 0) {
```

Comment:
Earlier we were using the length of inputs, which was an object. 
In the case of repeatables, `Object.keys(input.inputs).length`, will return the length of array instead.
In order to get the length of inputs in input-plugin, I hard-coded to get the length of inputs.